repository-data/application/src/main/resources/hcm-content/content/documents/administration/labels/modules/dev-module.yaml

```
@@ -19,22 +19,17 @@
-        viewToggle-results]
-      resourcebundle:messages: [Visit Website, Select features, View results, Results,
-        Select at least one feature, results found, No matched found — select fewer
-          options, No matched found — select fewer options, Select at least one feature,
-        results found, Results, Select features, View results]
+      resourcebundle:keys: [cta, alert-no-matches, alert-no-selections, alert-result-count,
+        results-heading, viewToggle-features, viewToggle-results, next-steps-text,
+        submit-url, form-title]
+      resourcebundle:messages: [Visit Website, No matched found — select fewer options,
+        Select at least one feature, results found, Results, Select features, View
+          results, placeholder, 'https://api.visitscotland.com/dev/submit/ssg/obs/submit',
+        CMS form title]
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 32 but found 8

(indentation)